config/theme-tabler.php

```diff
@@ -19,22 +19,10 @@
      * Simple pick a layout and let Backpack decide the best look for it.
      * No extra step is required.
      *
-     * Where should the menu be positioned?
+     * Possible values: horizontal, horizontal_dark, horizontal_overlap, vertical,
+     * vertical_dark, vertical_transparent, right_vertical, right_vertical_dark, right_vertical_transparent
      */
     'layout' => 'horizontal',
-    // horizontal
-    // horizontal_dark
-    // horizontal_overlap
```

Comment:
This is a small one, but why do progress widgets in the `horizontal_overlap` layout have this 1px left border? 👀 it's weird:
![CleanShot 2023-04-19 at 11 39 15@2x](https://user-images.githubusercontent.com/1032474/233019028-5e527781-7f38-4080-8841-49204afa4b87.png)

I really hope widgets don't have any conditionals (**if layout X, then look a certain way**).